CMakeLists.txt

```diff
@@ -421,6 +421,24 @@ if (FMT_TEST)
   add_subdirectory(test)
 endif ()
 
+if(EMSCRIPTEN)
+    set(FMT_WASM_SOURCES
+        ${FMT_SOURCES}
+        ${FMT_HEADERS}
+    )
```

Comment:
I don't think we need `FMT_WASM_SOURCES`, let's just use `${FMT_SOURCES}` and `${FMT_HEADERS}` in the target below.